lib/ansible/runner.rb

```diff
@@ -300,6 +300,19 @@ def fetch_galaxy_roles(playbook_or_role_args)
         Ansible::Content.new(playbook_dir).fetch_galaxy_roles
       end
 
+      def python_env
+        python3_modules_path = "/var/lib/awx/venv/ansible/lib/python3.6/site-packages/"
+        python2_modules_path = "/var/lib/awx/venv/ansible/lib/python2.7/site-packages/"
```

Comment:
Are the locations for these different on `upstream`/`downstream`?

Or maybe the proper questions is if we have to install things differently on `upstream`/`downstream`, if if that affects the location of these packages.  Does it make sense that these are configurable vars?

Not a big deal, but something to consider and possibly do in a followup.

Reply:
Okay, just looked at https://github.com/ManageIQ/manageiq-appliance-build/pull/340 and it seems like most of my questions are answered by that.